File: blogmanager/blogmanagerweb/src/main/webapp/js/service/permissionService.js
permissionApp.service('permissionService',function($http){
   this.findPage=function(page,rows){
       return $http.get('../permission/list.do?page='+page+"&rows="+rows);
   }

   this.addPermission=function(entity){
       return $http.post('../permission/add.do',entity);
   }
});

File: blogmanager/blogmanagerservice/src/main/java/com/blog/service/PermissionService.java
package com.blog.service;


import com.blog.common.pojo.EUDataGridResult;
import com.blog.common.pojo.Result;
import com.blog.pojo.TbPermission;
import com.blog.pojo.TbRole;

public interface PermissionService {
    EUDataGridResult gerPermissionList(Integer page, Integer rows);
    Result addPermission(TbPermission permission);
}
